6.0_RELEASE_NOTES.md

```diff
@@ -8,12 +8,45 @@ to a minimal. *(placeholder: talk about deprecated vars cleanup in a future
 release)*.
 
 ##  Webpack tree shaking!
+
+We are excited to announce officially supporting tree shaking. This means that
+assuming you use Webpack 4 or above, you can now import things directly from the
+`office-ui-fabric-react` package without memorizing top level imports. Webpack
+will tree shake out everything you don't use, which creates much smaller bundles!
+
+In order to make tree shaking work, webpack must consume ES6 modules. And since
+current Fabric consumers use `/lib/` imports, we needed to adjust the output folders
+within the repo:
+
+`/lib/` - ES6 modules (was commonjs). The `/lib-es2015` folder no longer exists, so
+make sure to clean up aliases if you were using them.
+
+`/lib-commonjs` - If you refer to fabric components in unit tests via Jest, you
+may need to alias the `/lib/` imports to to `/lib-commonjs`. Or use Babel.
+
+Note: To resolve commonjs modules in Jest, you can use this config blurb in your
+`jest.config.js`:
+
+```js
+  moduleNameMapper: {
+    'office-ui-fabric-react/lib/': 'office-ui-fabric-react/lib-commonjs/'
+  },
+```
+
+`lib-amd` - AMD modules. If you still need AMD, it is important to note another
```

Comment:
We should update `ghdocs/BestPractices/Advanced.md` as well with this

Reply:
done